examples/plot_inverse.py

```diff
@@ -0,0 +1,143 @@
+# Ephypype creates for us a pipeline which can be connected to these
+# nodes we created. The inverse solution pipeline is implemented by the
+# function :func:`ephypype.pipelines.preproc_meeg.create_pipeline_source_reconstruction`,  # noqa
+# thus to instantiate the inverse pipeline node, we import it and pass our
+# parameters to it.
+# The inverse pipeline contains three nodes that wrap the MNE Python functions
+# that perform the source reconstruction steps.
+#
+# In particular, the three nodes are:
+#
+# * :class:`ephypype.interfaces.mne.LF_computation.LFComputation` compute the
+#   Lead Field matrix
+# * :class:`ephypype.interfaces.mne.Inverse_solution.NoiseCovariance` computes
+#   the noise covariance matrix
+# * :class:`ephypype.interfaces.mne.Inverse_solution.InverseSolution` estimates
+#   the time series of the neural sources on a set of dipoles grid
+
+from ephypype.pipelines.fif_to_inv_sol import create_pipeline_source_reconstruction  # noqa
+inv_sol_workflow = create_pipeline_source_reconstruction(
+    data_path, sbj_dir, spacing=spacing, inv_method=inv_method, parc=parc,
+    noise_cov_fname=noise_cov_fname)
+
+###############################################################################
+# We then connect the nodes two at a time. First, we connect the two outputs
+# (subject_id and session_id) of the infosource node to the datasource node.
+# So, these two nodes taken together can grab data.
+
+main_workflow.connect(infosource, 'subject_id', datasource, 'subject_id')
+main_workflow.connect(infosource, 'session_id', datasource, 'session_id')
+
+###############################################################################
+# Similarly, for the inputnode of the preproc_workflow. Things will become
+# clearer in a moment when we plot the graph of the workflow.
+
+main_workflow.connect(infosource, 'subject_id',
+                      inv_sol_workflow, 'inputnode.sbj_id')
+main_workflow.connect(datasource, 'raw_file',
+                      inv_sol_workflow, 'inputnode.raw')
+
+###############################################################################
+# To do so, we first write the workflow graph (optional)
+
+main_workflow.write_graph(graph2use='colored')  # colored
+
+###############################################################################
+# and visualize it. Take a moment to pause and notice how the connections
+# here correspond to how we connected the nodes.
+
+from scipy.misc import imread  # noqa
+import matplotlib.pyplot as plt  # noqa
+img = plt.imread(op.join(data_path, src_reconstruction_pipeline_name, 'graph.png'))  # noqa
+plt.figure(figsize=(8, 8))
+plt.imshow(img)
+plt.axis('off')
+
+###############################################################################
+# Finally, we are now ready to execute our workflow.
+
+main_workflow.config['execution'] = {'remove_unnecessary_outputs': 'false'}
+
+# Run workflow locally on 3 CPUs
+main_workflow.run(plugin='MultiProc', plugin_args={'n_procs': 3})
```

Comment:
this will crash on someone's computer who doesn't have 3 cores. Use `'n_procs': 1`